Postman Test Suite/Ed-Fi ODS-API Profile Test Suite.postman_collection.json

```diff
@@ -11597,6 +11597,710 @@
+									"options": {
+										"raw": {
+											"language": "json"
+										}
+									}
+								},
+								"url": {
+									"raw": "{{ApiBaseUrl}}/data/v3/ed-fi/localEducationAgencies/{{known:localEducationAgencyId:A}}",
+									"host": [
+										"{{ApiBaseUrl}}"
+									],
+									"path": [
+										"data",
+										"v3",
+										"ed-fi",
+										"localEducationAgencies",
+										"{{known:localEducationAgencyId:A}}"
+									]
+								}
+							},
+							"response": []
+						},
+						{
+							"name": "The Write content type only excludes certain properties from a base class child collection",
+							"event": [
+								{
+									"listen": "test",
+									"script": {
+										"exec": [
+											"pm.test(\"The response should indicate success\", () => {\r",
+											"    pm.expect(pm.response.code).to.equal(204);\r",
+											"});\r",
+											"\r",
+											"// Made it a function, to avoid async issues\r",
+											"function getKnownResource (callback) {\r",
+											"    pm.sendRequest({\r",
+											"        url: pm.environment.get(\"ApiBaseUrl\") + `/data/v3/ed-fi/localEducationAgencies/` + pm.environment.get(\"known:localEducationAgencyId:B\"),\r",
+											"        method: \"GET\",\r",
+											"        header: {\r",
+											"            \"Authorization\": \"bearer \" + pm.environment.get(\"AccessToken_255901\"),\r",
+											"            \"Content-Type\": \"application/json\"\r",
+											"        }\r",
+											"    }, callback);\r",
+											"}\r",
+											"\r",
+											"getKnownResource((error, response) => {\r",
+											"    Validate(response.json());\r",
+											"});\r",
+											"\r",
+											"function Validate(jsonResponse) {\r",
+											"    const originalEntityAddressesCollection = pm.environment.get(\"known:localEducationAgency:B\")[\"addresses\"][0];\r",
+											"    const updatedEntityAddressesCollection = jsonResponse[\"addresses\"][0];\r",
+											"    const contextualPrimaryKeys = pm.environment.get(\"known:educationOrganizationAddress:PrimaryKeys\");\r",
+											"    const profileExcludeKeys = [\r",
+											"        \"doNotPublishIndicator\"\r",
+											"    ]\r",
+											"            console.log(originalEntityAddressesCollection)\r",
+											"            console.log(updatedEntityAddressesCollection)\r",
+											"    pm.test(\"The only values not changed on the entity model's base class addresses collection items should be the contextual primary key values, and the explicitly excluded properties\",\r",
+											"    () => {\r",
+											"        // Traverse all properties\r",
+											"        for (const [key, value] of Object.entries(originalEntityAddressesCollection)) {\r",
```

Comment:
This is a nitpic and it's on me too but we're using a lot of `Object.entries` when we could really be using `Object.keys`.